src/shared_memory_manager.cc

```diff
@@ -297,8 +297,8 @@ CheckCudaSharedMemoryRegionSize(
     return TRITONSERVER_ErrorNew(
         TRITONSERVER_ERROR_INVALID_ARG,
         std::string(
-            "failed to register CUDA shared memory region '" + name +
-            "': register size exceeds CUDA shared memory region size")
+            "failed to register shared memory region '" + name +
```

Comment:
failed to register `CUDA` shared memory region

Reply:
why did we resolve this conversation? It seems like adding CUDA here is still a good idea?

Reply:
I think after the discussion with Neelay, we think that it's better to remove the `CUDA` keyword to expose as little information as possible to the client: https://github.com/triton-inference-server/server/pull/7178#discussion_r1589740725